library/core/tests/mem.rs

```diff
@@ -137,3 +137,16 @@ fn assume_init_good() {
 
     assert!(TRUE);
 }
+
+#[test]
+#[cfg(not(bootstrap))]
```

Comment:
The test should pass with and without bootstrap, shouldn't it? `mem::forget` is const-stable already...

Reply:
I would have thought so too, but https://github.com/rust-lang/rust/runs/1548179565
![image](https://user-images.githubusercontent.com/18526288/102063231-a6ce1800-3ded-11eb-9ea2-0418bca823ad.png)

Reply:
Ah, that's probably because of the missing intrinsic implementation in the bootstrap compiler. Good point.